service/internal/cloudconfig/tccpn.go

```diff
@@ -354,6 +354,10 @@ func (t *TCCPN) newTemplate(ctx context.Context, obj interface{}, mapping hamast
 			HighAvailability:    multiMasterEnabled,
 			NodeName:            key.ControlPlaneEtcdNodeName(mapping.ID),
 		}
+		// we need to explicitly set InitialCluster for single master, since k8scc qhas different config logic which does nto work for AWS
+		if !multiMasterEnabled {
+			params.Etcd.InitialCluster = fmt.Sprintf("%s=https://%s.%s:2380", key.ControlPlaneEtcdNodeName(mapping.ID), key.ControlPlaneEtcdNodeName(mapping.ID), key.TenantClusterBaseDomain(cl))
```

Comment:
Just for my understanding, what happens when this value is not explicitly set? (In the multi master case)

Reply:
k8scc is checking the value and if its empty i will fill it with its internal logic, but resulting in issue inc ase of single master in AWS